main.py

```diff
@@ -1,19 +1,23 @@
 import pygame
 from tanks.constants import *
 import tanks.grid as grid
+from tanks.load import load_level, create_random_level
```

Comment:
```suggestion
from tanks.load import load_level
```
Этот метод не нужен, так как играбельную карту он все равно не сгенерирует. Для тестов лучше вручную сделать тестовые уровни.